redbot/__main__.py

```diff
@@ -138,6 +138,11 @@ async def _edit_prefix(red, prefix, no_prompt):
             if not prefixes:
                 print("You need to pass at least one prefix!")
                 continue
+            if prefix.startswith("/"):
```

Comment:
`prefix` is false-like here, if it is a string it will always be empty. This will not have the desired effect, and may error (the function isn't typehinted so I don't know for certain if it would be an empty string or `None`).